### DISCUSSION TITLE
Add support for changing slider bar values via mouse scroll wheel

### DISCUSSION THREAD

**Discussion by artun42** [ppy/osu#13135] [Ideas]
Add an ability to scrolling on slider thing by mouse wheel. Might be useful by setting accurate offset to audio.

**Comment by bdach** [comment]
You can use arrow keys already, but scroll wheel might also be nice... although it could be a little wonky in case of slider bars inside of scrolling containers.

I guess in that case you could say that if the mouse is hovered over a slider bar, it should increment/decrement it, and if it's *not* hovered over a slider, then it should scroll?

**Comment by artun42** [comment]
Yes, that's exactly what I meant.

**Comment by Shompi** [comment]
Also maybe make it so it only changes the slider value when the screen or panel is not scrolling (full stop) plus on hover to prevent unexpected or non on purpose slider value changes.

**Comment by peppy** [comment]
We cannot make it based on hover. It will interrupt vertical scrolling.

If we're going to support this, sliders need to have a focus mode, where you click them first to select then you can use the wheel to adjust.

Or they should handle horizontal scroll only.

**Comment by LastExceed** [comment]
Or require a modifier key to be held
